sdks/java/testing/nexmark/build.gradle

```diff
@@ -102,6 +109,39 @@ if (shouldProvideSpark) {
   }
 }
 
+def getNexmarkArgs = {
+  def nexmarkArgsStr =  project.findProperty(nexmarkArgsProperty) ?: ""
+  def nexmarkArgsList = new ArrayList<String>()
+  Collections.addAll(nexmarkArgsList, nexmarkArgsStr.split())
+
+  if (isDataflowRunner) {
+    if (isDataflowRunnerV2) {
+      nexmarkArgsList.add("--experiments=beam_fn_api,use_unified_worker,use_runner_v2,shuffle_mode=appliance")
```

Comment:
I would suggest you use `shuffle_mode=service`. Currently we just focus on runnver v2 + shuffle service.

Reply:
Same for Java8 setup